src/auth/handlers/oauth.handelr.ts

```diff
@@ -0,0 +1,93 @@
+import { HttpService } from '@nestjs/axios';
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Session } from 'src/entities/session.entity';
+import { Repository } from 'typeorm';
+import { User } from '../../entities/user.entity';
+import { EmailHandler } from './mail.handler';
+import { SessionHandler } from './session.handler';
+@Injectable()
+export class OauthHandler {
+  constructor(
+    @InjectRepository(User) 
```

Comment:
don't inject repository, use can use auth service

Reply:
should I move authUser function to auth service?